lib/KeyboardAwareScrollView.js

```diff
@@ -25,6 +25,7 @@ const KeyboardAwareScrollView = React.createClass({
       <ScrollView
         ref='_rnkasv_keyboardView'
         keyboardDismissMode='interactive'
+        keyboardShouldPersistTaps={true}
```

Comment:
This change isn't needed because you can pass `keyboardShouldPersistTaps` through `KeyboardAwareScrollView` props.